### PR TITLE
🎨 [Frontend] Handle LongRunningTask errors better

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Display error message provided by the backend
- [x] Show "No message" text if the flash message is empty

ErrorMessage

## Related issue/s

follows https://github.com/ITISFoundation/osparc-simcore/pull/6444

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/data/Resources.js [hunk 1]
```diff
@@ -1337,6 +1337,9 @@ qx.Class.define("osparc.data.Resources", {
 
           if ([404, 503].includes(status)) {
             message += "<br>Please try again later and/or contact support";
+          } else if (status == 500) {
+            console.error(message);
+            message += "<br>Please contact support";
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/data/Resources.js:1342]: All 500 now have a proper message from the backend. I would not add "contact support"

**Review by GitHK — APPROVED** [review]
👍

**Review by pcrespov — COMMENTED** [review]
In the video attached to the PR, are you using the latest version of the stack, including https://github.com/ITISFoundation/osparc-simcore/pull/6444 ? In theory, after that commit, **unhandled exceptions are transformed into 500** with json-response as
```json
{
  "error": {
      "status": 500, 
      "message": "We apologize for the inconvenience. Our team has recorded the issue and is working to resolve it as quickly as possible. Thank you for your patience [OEC: 1234]"
   }
}
```

and on the web-server logs you should get a comprehensive log with details of the exception including the context

**Comment by odeimaiz** [comment]
> In the video attached to the PR, are you using the latest version of the stack, including #6444 ? In theory, after that commit, **unhandled exceptions are transformed into 500** with json-response as
> 
> ```json
> {
>   "error": {
>       "status": 500, 
>       "message": "We apologize for the inconvenience. Our team has recorded the issue and is working to resolve it as quickly as possible. Thank you for your patience [OEC: 1234]"
>    }
> }
> ```
> 
> and on the web-server logs you should get a comprehensive log with details of the exception including the context

Nope, the commit wasn't in yet.

**Review by jsaq007 — APPROVED** [review]

**Pull request merged**